lms/static/scripts/frontend_apps/services/client-rpc.js

```diff
@@ -38,8 +38,9 @@ import { JWT } from '../utils/jwt';
  *
  * @typedef ContentInfoItem
  * @prop {string} title - Title of the current article, chapter etc.
- * @prop {string} containerTitle - Title of the journal issue, book etc. which
- *   contains the current work
+ * @prop {string} [subtitle]
```

Comment:
The typing in this module defines what the LMS frontend sends to the client. It's different from , but moving toward, the "schema" of the proxy API as typed in `api-types`. 

These changes add a `container` object, but leave the `containerTitle` in place until changes can be made on the client side (otherwise we'd break the client).